js/library/jquery.popup.js

```diff
@@ -133,6 +133,11 @@ Copyright 2007 Chris Wanstrath [ chris@ozmm.org ]
       });
       $('#'+settings.popupId).show();
 
+      $('#'+settings.popupId+' .Body').css({
+         'max-height': pagesize[3] * .8,
+         'overflow-y': 'scroll'
```

Comment:
`overflow-y: scroll` **always** shows a scroll bar, even if it is not needed (see login dialog as an example)
`overflow-y: auto` would be better